src/models/network.js

```diff
@@ -46,8 +46,8 @@ Network.prototype.export = function() {
 		"commands"
 	]);
 	network.nick = (this.irc || {}).me;
-	network.join = _.pluck(
-		_.where(this.channels, {type: "channel"}),
+	network.join = _.map(
+		_.filter(this.channels, {type: "channel"}),
```

Comment:
Same as above, I'd rather see:

```js
this.channels.filter(function (channel) {
  return channel.type === "channel";
}).map(function (channel) {
  return channel.name;
}).join(",");
```

Yes, it's a few extra chars, but this gets us as close as the language as possible so I think it's a win! Plus we can even improve it with `.reduce()` if necessary (but I don't believe it is)!